docs/source/cuda-reference/kernel.rst

```diff
@@ -175,7 +175,7 @@ Synchronization and Atomic Operations
     indices for indexing into multiple dimensional arrays. The number of element
     in ``idx`` must match the number of dimension of ``array``.
 
-    Returns the value of ``array[idx]`` before the storing the new value.
```

Comment:
I've removed a few extraneous `the` from this file, which aren't strictly speaking relevant to this PR.  I can revert if preferred.

Reply:
That's great, many thanks for a nice little tidy-up!